src-docs/src/views/tables/mobile/mobile_section.js

```diff
@@ -17,6 +17,7 @@ const exampleItem = `{
     render: (item) => (<span>{item.firstName} {item.lastName}</span>), // Custom renderer for mobile view only
     header: false,   // Won't show inline header in mobile view
     fullWidth: true, // Forces 100% width of the cell
+    width: '100%' // Forces 100% width of the cell
```

Comment:
While we don't want to break anyone's usage of `fullWidth`. I think it's safe to remove in **favor** of `width`.

```suggestion
    width: '100%' // Applies a specific width
```